src/xrEngine/xr_input_xinput.cpp

```diff
@@ -138,11 +138,11 @@ _action actions[] = {
 
 _keyboard keyboards[] =
 {
-    { "mouse1",                 MOUSE_1,                         "Mouse 1" },
-    { "mouse2",                 MOUSE_2,                         "Mouse 2" },
-    { "mouse3",                 MOUSE_3,                         "Mouse 3" },
-    { "mouse4",                 MOUSE_4,                         "Mouse 4" },
-    { "mouse5",                 MOUSE_5,                         "Mouse 5" },
+    { "mouse1",                 MOUSE_1,                         "LMB" },
+    { "mouse3",                 MOUSE_3,                         "MMB" },       // This is not a mistake cause init algorithm was changed.
```

Comment:
It won't break something, yes?
![image](https://user-images.githubusercontent.com/2989212/56686763-2a556580-66ee-11e9-84f9-b8e3bd2f174c.png)

Reply:
I just give mouse buttons correct user friendly name. And yeap, I used some "hack" because mouse buttons in SDL and in vanilla xray have different positions in "key list". If you change this, I think the game will show Middle button like Right button, and vice versa.